components/brave_new_tab_ui/data/colors.ts

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const solidColorsForBackground: string[] = [
```

Comment:
Optional: We could also add the default color here. Might not be worth it though